.envrc

```diff
@@ -0,0 +1,8 @@
+NODEVERSION=16
```

Comment:
Shouldn't we default to 18? @RunDevelopment

Reply:
Probably. We only support >=16.14 anyway.